### PR TITLE
fixing PI for reporting

### PR DESCRIPTION
**Pull request by lmichael107**

### CODE REVIEW

File: projects/COVID19_FoldingAtHome.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ FieldOfScience: Biochemistry
 ID: '661'
 Name: COVID19_FoldingAtHome
 Organization: University of Nebraska-Lincoln
```
Comment (brianhlin) [inline projects/COVID19_FoldingAtHome.yaml:6]: Should we also update the org?

Comment (lmichael107) [reply projects/COVID19_FoldingAtHome.yaml:6]: yes! Did we ever agree on the "Multi-Institutional" option? See: https://foldingathome.org/about/the-foldinghome-consortium/

**Comment by lmichael107** [comment]
We should also fix Organization? I was just asking @brianhlin about "Multi-Institutional", given https://foldingathome.org/about/the-foldinghome-consortium/

**Comment by brianhlin** [comment]
I think we could just use `Folding@Home Consortium` like we do https://github.com/opensciencegrid/topology/blob/master/projects/LIGO.yaml#L6

File: projects/COVID19_FoldingAtHome.yaml (hunk 1 above)
Comment (brianhlin) [inline projects/COVID19_FoldingAtHome.yaml:6]:
```suggestion
Organization: Folding@Home Consortium (FAHC)
```

Comment (djw8605) [reply projects/COVID19_FoldingAtHome.yaml:6]: Just put it now, we can fix it later.  Lets get this merged.

**Comment by lmichael107** [comment]
looks good!

**Pull request merged**